### ISSUE TITLE
Cannot see console.logs

### ISSUE THREAD

**Issue by kraftwer1** [airportyh/testem#562]
I use a lot of console.logs() when I write tests. Unfortunately, Testem doesn't show it to me as you can see in screenshot 1. I'll have to resize my terminal to make them visible, see screenshot 2.

Any idea how I can fix the layout in my terminal? I'm using OSX 10.10 and it happens in Terminal.app as well as in iTerm2.app.

![bildschirmfoto 2015-05-22 um 16 04 05](https://cloud.githubusercontent.com/assets/2146436/7772085/4a22da4e-009c-11e5-85d9-9862b1c19c05.png)
![bildschirmfoto 2015-05-22 um 16 04 30](https://cloud.githubusercontent.com/assets/2146436/7772086/4a3b1c4e-009c-11e5-85da-c21dfa8096de.png)

**Comment by airportyh** [comment]
It's possible that a lot of blank lines or undefined values are being
printed via console.log. Could you verify if that is the case or not?

Toby

**Comment by kraftwer1** [comment]
I don't think so, since the phrase _"1 tests complete."_ is always exactly in the middle (no matter how the window is resized), it looks more like a layout bug...

**Comment by kraftwer1** [comment]
I have played around with the code a little bit and I've found the following:

*lib/dev/ui/split_log_panels.js:163*

```
var line = midLine + tabs.TabStartLine + tabs.TabHeight - 1
```

...when I overwrite this like:

```
var line = 1
````

...the layout looks correct (apart from the missing line around the tab above), see:

![bildschirmfoto 2015-05-26 um 15 52 38](https://cloud.githubusercontent.com/assets/2146436/7813994/7180f528-03bf-11e5-9e2f-434059362d06.png)
